content/en/observability_pipelines/sources/azure_event_hubs.md

```diff
@@ -0,0 +1,176 @@
+
+1. In the Event Hub you created, navigate to **Settings** > **Shared access policies**.
+1. Click **+ Add**.
+1. Enter a policy name (for example, `DatadogKafkaPolicy`).
+1. Select the **Manage** checkbox, which should automatically select the **Send** and **Listen** checkboxes.
+1. Click **Create**.
+1. The **Primary Key** and **Primary connection string** is needed for Kafka authentication when you set up the Observability Pipeline's Kafka source.
+
+### Set up diagnostic settings
+
+1. Configure Azure resources (for example, VMs, App Services) or subscription-level activity logs to stream logs to the Event Hub.
+1. For resources:
+    1. Navigate to the resource and then to **Monitoring** > **Diagnostic settings**.
+    1. Click **+ Add diagnostic setting**.
+    1. Select log categories you want (for example, AuditLogs, SignInLogs for Microsoft Entra ID).
+    1. In **Destination details**:
+        1. Check the **Stream to an event hub** box.
+        1. Select the namespace and Event Hub (`datadog-topic`).
+    1. Click **Save**.
+1. For activity logs:
+    1. Navigate to **Microsoft Entra ID** > **Monitoring** > **Audit logs** > **Export Data Settings**.
+    1. Check the **Stream to the Event Hub** box.
+1. Repeat for each region. Logs must stream to Event Hubs in the same region.
+
+### Configure Kafka-compatible connection for the Event Hub
+
+Azure Event Hubs exposes a Kafka endpoint at `NAMESPACE.servicebus.windows.net:9093`, which Observability Pipelines uses as the Kafka source.
+
+#### Get the Kafka endpoint
+
+1. In the Azure Portal, navigate to your Event Hubs Namespace (for example, `myeventhubns`).
+1. On the **Overview** page, under the **Essentials** section, locate the **Host name** or **Fully Qualified Domain Name (FQDN)**. It is in the format: `<NAMESPACE>.servicebus.windows.net` (for example, `myeventhubns.servicebus.windows.net`).
+1. Append the Kafka port `:9093` to form the Bootstrap Servers value: `<NAMESPACE>.servicebus.windows.net:9093`.
+    - For example, if your namespace is `myeventhubns`, the Bootstrap Servers is `myeventhubns.servicebus.windows.net:9093`.
+    - You need this information when you set up the Observability Pipelines Kafka source.
+
+#### Set up authentication
+
+1. Azure Event Hubs uses SASL_SSL with the PLAIN mechanism for Kafka authentication.
+1. The connection string is formatted for Observability Pipelines:
+    ```
+    Username: $$ConnectionString
+    Password: Endpoint=sb://<NAMESPACE>.servicebus.windows.net/;SharedAccessKeyName=<PolicyName>;SharedAccessKey=<Key>
+    ```
+
+## Set up a pipeline with the Kafka source
+
+Select your platform.
+
+{{< tabs >}}
+{{% tab "Kubernetes" %}}
+1. Navigate to [Observability Pipelines](https://app.datadoghq.com/observability-pipelines).
+1. Select the Kafka source.
+    1.  In the **Group ID** field, specify or create a unique consumer group (for example, `datadog-consumer-group`).
+    1.  Enter `datadog-topic` in the **Topics** field.
```

Comment:
Should this say something like "or the topic you configured for your Event Hub"? We suggest this value above, but it may not be the value the customer uses right?